test.sh

```diff
@@ -0,0 +1,12 @@
+set -x
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]